consultation_analyser/support_console/views.py

```diff
@@ -13,3 +18,34 @@ def support_home(request: HttpRequest):
 def sign_out(request: HttpRequest):
     logout(request)
     return redirect("/")
+
+
+@staff_member_required
+def show_consultations(request: HttpRequest) -> HttpResponse:
```

Comment:
In all other frameworks (Rails, Laravel, .NET etc), these plural `show the list of things` controller methods are called "index" — is it worth adopting that convention here? So by convention there would be a "show" and "index" view function in each set of views corresponding to a given entity

Reply:
So would this be `index_consultations`?

I'm not sure if there are any naming conventions with Django (at least that I'm aware of) - so happy to go with any conventions (and better to have consistency).

Reply:
Also, I don't know why I named them `show_consultation` and `show_consultations` because that's really confusing.

Reply:
it would be 

```
from consultation_analyser.support_console.views import consultations

consultations.index
consultations.support
```

Reply:
I'll merge this PR and then refactor the views/rename in a separate PR.